### ISSUE TITLE
x/pkgsite: update subrepo list

### ISSUE THREAD

**Issue by seankhliao** [golang/go#56482]
### What is the URL of the page with the issue?

https://pkg.go.dev/golang.org/x

Currently the sub repo page lists the following repos

> [benchmarks](https://pkg.go.dev/golang.org/x/benchmarks) — benchmarks to measure Go as it is developed.
[blog](https://pkg.go.dev/golang.org/x/blog) — [blog.golang.org](https://blog.golang.org/)'s implementation.
[build](https://pkg.go.dev/golang.org/x/build) — [build.golang.org](https://build.golang.org/)'s implementation.
[crypto](https://pkg.go.dev/golang.org/x/crypto) — additional cryptography packages.
[debug](https://pkg.go.dev/golang.org/x/debug) — an experimental debugger for Go.
[image](https://pkg.go.dev/golang.org/x/image) — additional imaging packages.
[mobile](https://pkg.go.dev/golang.org/x/mobile) — experimental support for Go on mobile platforms.
[net](https://pkg.go.dev/golang.org/x/net) — additional networking packages.
[perf](https://pkg.go.dev/golang.org/x/perf) — packages and tools for performance measurement, storage, and analysis.
[pkgsite](https://pkg.go.dev/golang.org/x/pkgsite) — home of the pkg.go.dev website.
[review](https://pkg.go.dev/golang.org/x/review) — a tool for working with Gerrit code reviews.
[sync](https://pkg.go.dev/golang.org/x/sync) — additional concurrency primitives.
[sys](https://pkg.go.dev/golang.org/x/sys) — packages for making system calls.
[text](https://pkg.go.dev/golang.org/x/text) — packages for working with text.
[time](https://pkg.go.dev/golang.org/x/time) — additional time packages.
[tools](https://pkg.go.dev/golang.org/x/tools) — godoc, goimports, gorename, and other tools.
[tour](https://pkg.go.dev/golang.org/x/tour) — [tour.golang.org](https://tour.golang.org/)'s implementation.
[exp](https://pkg.go.dev/golang.org/x/exp) — experimental and deprecated packages (handle with care; may change without warning).

"blog" should be removed, the repo has been cleared out
"mod", "oauth2", "term", and "vuln" seem to be suitable candidates for adding to the list
"exp" could be moved up to follow alphabetical order?

---

Could a link to this page be added to the footer under Packages / Standard library?

**Comment by findleyr** [comment]
Suggested changes make sense to me. Not sure whether oauth2 and term were intentionally omitted, since they have existed for some time. mod and vuln are new enough that I suspect they just post-dated the original list.

CC @jamalc, who appears to have some context on this page.

**Comment by jamalc** [comment]
We should make this page more accessible, a link could fit nicely in the new navbar and the footer. I believe we copied the list from godoc for #47281. #37265 mentions that the list was intentionally not exhaustive but I'm not sure what is the threshold for noteworthy.

**Comment by AGWA** [comment]
If the list is intentionally not exhaustive, I would suggest noting this on the page.  I ended up reinventing the wheel on some modules-related stuff because I didn't know that `golang.org/x/mod` existed; I would have looked harder if I knew the list wasn't exhaustive.

**Comment by findleyr** [comment]
It took me a while to get to this, sorry.

I've made most of the recommended changes here. I also included x/website in the list.

@jamalc you mentioned making this page more accessible. It seemed OK to me, though I didn't try it out with a screen reader. Can we close this issue and handle accessibility more holistically in a separate issue? (I think we have plans to do a more widespread accessibility analysis)
